src/service/ActivityEventService.ts

```diff
@@ -3,12 +3,28 @@ import { ActivityEvent } from "../model/ActivityEvent"
 import { ActivityEventRepository } from "../repository/ActivityEventRepository"
 import { SecurityContext, ActionContext, _verify } from "./Security"
 import jsonata from "jsonata"
-
+import Ajv from 'ajv';
+import api_schema from './api_schema.json'
```

Comment:
Please do not copy-paste the schema and import it this way. Take a look at how the `app.ts` file loads the schema and insert a hook there to grab the cached version.